### PR TITLE
feat: add `live_reload` to default `common_site_config`

### PR DESCRIPTION
**Pull request by pruthvi145**
## Changes Made
 - add `live_reload` to default_config
 - add patch to set `live_reload` for existing bench instances

---
**Related PR:** https://github.com/frappe/frappe/pull/14238

### CODE REVIEW

File: patches.txt [hunk 1]
```diff
@@ -1 +1 @@
-bench.patches.v3.deprecate_old_config
+bench.patches.v3.deprecate_old_config
```
Comment (gavindsouza) [inline patches.txt]: I think we can get rid of this file altogether. Will leave no scope for confusion

**Review by gavindsouza — COMMENTED** [review]
This will add the `live_reload` to the common site conf for benches of all Frappe versions. I hope this doesn't break or cause unwanted behaviour for someone else's apps :')

Acknowledging that risk, LGTM.

**Comment by gavindsouza** [comment]
I'm not sure why Travis hasn't finished running. @pruthvi145 Push another commit to re-trigger it? Perhaps drop the `patches.txt` file?

File: patches.txt (hunk 1 above)
Comment (pruthvi145) [reply patches.txt]: Done

**Comment by sagarvora** [comment]
> I hope this doesn't break or cause unwanted behaviour for someone else's apps :')

Doesn't `common_site_config` always come with this caveat :smile:
Although, we could do something like this:
```python
frappe_branch = get_current_branch("frappe", bench_path)
if frappe_branch.startswith(("version-11", "version-12", "version-13")):
    return
```

I don't think we should though. If and when they do upgrade to an eligible version, we want the config to be set.

**Review by gavindsouza — APPROVED** [review]

**Pull request merged**